src/main/java/by/exadel/internship/service/impl/FileServiceImpl.java

```diff
@@ -45,18 +52,54 @@ private String uploadFile(byte[] file, String fileName){
         log.info("Try to upload file to cloud");
         BlobId blobId = BlobId.of(bucketName, fileName);
         BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("media").build();
-        Credentials credentials = null;
+
+        Credentials credentials = getCredentialsByJSON();
+
+        Storage storage = StorageOptions.newBuilder().setCredentials(credentials).build().getService();
+        storage.create(blobInfo, file);
+        log.info("File was uploaded to cloud");
+        return fileName;
+    }
+
+    @Override
+    public String download(String fileName, String formLastName) {
+        log.info("Try to download file from cloud");
+        String destFileName = formLastName
```

Comment:
As I remember @SergeyKoval suggested to use first and last name + optional: internship name.
![image](https://user-images.githubusercontent.com/77664426/118350309-9c5fa800-b55e-11eb-9c25-c5701ddc4555.png)

Reply:
Fixed